test/e2e/framework.go

```diff
@@ -407,3 +407,9 @@ func kubectlExec(namespace string, podName, containerName string, args ...string
 	err := cmd.Run()
 	return stdout.Bytes(), stderr.Bytes(), err
 }
+
+// Wrapper function for ginkgo describe.  Adds namespacing.
+// TODO: Support type safe tagging as well https://github.com/kubernetes/kubernetes/pull/22401.
+func KubeDescribe(text string, body func()) bool {
+	return Describe(text+" [k8s.io] ", body)
```

Comment:
I'd rather have `[k8s.io] Variable Expansion should allow ...` than `Variable Expansion [k8s.io] should allow ...`.